ipysurfer/_utils/colormap.py

```diff
@@ -54,18 +13,26 @@ def _calculate_cmap(lim_cmap, alpha, ctrl_pts, scale_pts):
     alpha : float
         Alpha value to apply globally to the overlay. Has no effect with mpl
         backend.
-    ctrl_pts : tuple(float)
-        Color map control points.
-    scale_pts : tuple(float)
-        Data scale control points.
+    min : float
```

Comment:
probably best not to user "min" and "max" since these would overwrite core python functions. How about c_min and c_max? or `vmin` and `vmax` to be consistent with matplaotlib?

Reply:
yes, that would be better

Reply:
I will change the names to `fmin` and `fmax`, since these are the ones used in `MNE`.